### DISCUSSION TITLE
Grafana (pasword lost on synology)

### DISCUSSION THREAD

**Discussion by DjShoarma** [teslamate-org/teslamate#3204] [Q&A]
Hi there,

I've lost my password and login on Grafana (installed on my synology in a Docker Teslamate_grafana_1) to access my reports.
When I push forget password I don't get any recovery email.
I've installed it via a step plan, so not a very good programmer/user myself so what could I do to reset the Password on my synology docker?
Many thanks in advance.

**Comment by tpd-opitz** [comment]
I solved the problem using [this stackoverflow answer](https://stackoverflow.com/questions/54039604/what-is-the-default-username-and-password-for-grafana-login-page)
